### PR TITLE
Better towns representation, refs #399

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: udata/templates/territories/territory.html [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 {% set logo=territory.logo_url(external=True) or theme_static('img/placeholder_territory.png', external=True) %}
 {% set meta = {
-    'title': territory.name,
-    'description': territory.name + ' (' + territory.postal_string + ')',
+    'title': territory.town_repr,
+    'description': _('Datasets related to the town of ') + territory.town_repr + _(' with INSEE code ') + territory.code,
```
Comment (noirbizarre) [inline udata/templates/territories/territory.html]:
```python
_('Datasets related to the town of %(name)s with INSEE code %(code)s', name=territory.town_repr, code=territory.code)
```
But INSEE code is France specific, why not simply:
```python
_('Datasets related to the town of %(name)s', name=territory.town_repr)
```

Comment (davidbgk) [inline udata/templates/territories/territory.html]: I try to add as much infos as I can in the description for SEO :/

**Comment by davidbgk** [comment]
@noirbizarre with a new design:

capture d ecran 2016-03-23 a 11 16 22

**Comment by noirbizarre** [comment]
r+

**Pull request merged**